engine/core/src/test/java/org/pentaho/reporting/engine/classic/core/RotationTest.java

```diff
@@ -109,7 +111,7 @@ public void testXLS() throws ResourceException, IOException {
     }
   }
 
-  @Test
+  //@Test
```

Comment:
Let's write a jira ticket to fix these so they don't get forgotten.

Reply:
https://hv-eng.atlassian.net/browse/BACKLOG-40351 -- This is the ticket I have created to track these.